config/settings.yml

```diff
@@ -226,14 +226,19 @@ defaults: &defaults
   #####################
   # DIRECTOR SETTINGS #
   #####################
-  #bootstrap_bosh_director settings
-  bootstrap_bosh_director:
-    url: https://192.168.50.3:25555
+  directors:
+  - url: https://192.168.50.3:25555
+    primary: false        # is true if the director takes part in lifecycle operations
+    supportCreate: false  # is true if director is expected to 'create deployments'.
+    name: bootstrap-bosh
     username: admin
     password: admin
     skip_ssl_validation: true
     cpi: warden_cpi
-  director: &director
+  - &director
+    primary: true
+    supportCreate: false
```

Comment:
use snake casing in yml. it should be `support_create` (look at other names & keep it consistent)